suites/portal/fileUploadTest.js

```diff
@@ -0,0 +1,63 @@
+Feature('Data-Portal File Upload test with gen3-client CLI - https://ctds-planx.atlassian.net/browse/PXP-4542');
+
+// To be executed with GEN3_SKIP_PROJ_SETUP=true -- No need to set up program / retrieve access token, etc.
+
+const chai = require('chai');
+const {interactive, ifInteractive} = require('../../utils/interactive.js');
+const expect = chai.expect;
+const hostname = process.env.HOSTNAME;
+const profile = process.env.NAMESPACE
+
+Scenario('Create a folder in $vpc_name and generate a test file @manual', ifInteractive(
+    async(I) => {
+        const result = await interactive (`
+            1. Login into $vpc_name
```

Comment:
Is $vpc_name the admin vm?  Why do we need to login to the admin vm to run this test?

Reply:
Good point. There's no need to do it.
This can be executed by any local terminal. Will add another commit shortly.